programmers/p와y의-개수.js

```diff
@@ -0,0 +1,16 @@
+function solution(s){
+    var p = s.match(/p/gi);
+    var y = s.match(/y/gi);
+    
+    if (p != null && y != null ){
+        if (p.length != y.length)
+            return false;
+        else
+            return true;
+    }
+    else if (p == null && y == null )
```

Comment:
```suggestion
    else if (p === null && y === null )
```
자바스크립트에서 같다라는 표시는 === 이렇게 = 3개로 해주시는것이 좋아요